students/karisPlath/Lab0/Lab2.ino

```diff
@@ -0,0 +1,80 @@
+// pins!!
+const int apin=0;
+const int rLED=5;
+const int yLED=6;
+const int cLED=8;
+const int gLED=9;
+const int bLED=10;
+
+const float aref = 3.3;
+const int adc_max = 1023;
+const float temp_slope = 100;
+
+int val;
+float volt;
+float temp;
+
+
+void setup() {
+  // adding serial for debug reasons...
+  Serial.begin(9600);
+  analogReference(EXTERNAL);
+
+  // init pins
+  pinMode(apin, INPUT);
+  pinMode(rLED, OUTPUT);
+  pinMode(yLED, OUTPUT);
+  pinMode(cLED, OUTPUT);
+  pinMode(gLED, OUTPUT);
+  pinMode(bLED, OUTPUT);
+}
+
+void loop() {
+  // read sensor
+  val = analogRead(apin);
+
+  // convert to voltage
+  volt = val * aref / adc_max;
+  // volt = map(val, fromLow, formHigh, toLow, toHigh);
+  val = map(val, 0, adc_max, 0, aref);
```

Comment:
Remember we learned `map` uses integer math, which means your `val` will only take on a few possible values... 0, 1, 2, and 3.